go/home/home.go

```diff
@@ -100,7 +100,7 @@ func (h *Home) getToCache(ctx context.Context, markedViewed bool, numPeopleWante
 	return nil
 }
 
-func (h *Home) Get(ctx context.Context, markViewed bool, numPeopleWanted int) (ret keybase1.HomeScreen, err error) {
+func (h *Home) Get(ctx context.Context, markViewed bool, numPeopleWanted int, bustCache bool) (ret keybase1.HomeScreen, err error) {
```

Comment:
Can you explain why this needs to change? What about the previous system was insufficient for this use case?